src/gauging.jl

```diff
@@ -173,8 +167,9 @@ function symmetric_to_vidal_gauge(
 
   for e in edges(ψ)
     vsrc, vdst = src(e), dst(e)
-    pe = partitionedge(pψψ, NamedEdge((vsrc, 1) => (vdst, 1)))
-    bond_tensors[e], bond_tensors[reverse(e)] = only(mts[pe]), only(mts[pe])
+    pe = partitionedge(partitioned_itensornetwork(bpc), NamedEdge((vsrc, 1) => (vdst, 1)))
```

Comment:
Same comment as above about making `partitionedge` work for `BeliefPropagationCache`.

Also, is `NamedEdge` needed here?

Reply:
Sure. And yeah you're right `NamedEdge` is no longer needed.